### ISSUE TITLE
Combat Start timer missing for Vaelastrasz in BWL - Classic ERA

### ISSUE THREAD

**Issue by mimixy** [DeadlyBossMods/DBM-Classic#782]
Vael (2nd boss in BWL), doesn't have a combat start timer anymore. It doesn't create any LUA errors. It disapeared a few weeks ago. I've done it many times on many toons, and the timer hasn't showed up once on any toon. I've checked the DBM settings, and the option for the timer is on.
![Screenshot 2023-12-20 084129](https://github.com/DeadlyBossMods/DBM-Classic/assets/143043909/5e748258-98c0-4b32-9eb6-362f6412d9f2)

**Comment by MysticalOS** [comment]
Retest this in newest update going out soon please

**Comment by mimixy** [comment]
tested today, still no timer. all other timers are there in all boss fights as expected. this is the only missing timer
![WoWScrnShot_010424_141636](https://github.com/DeadlyBossMods/DBM-Classic/assets/143043909/efc1a0cb-6ac3-4a5b-afa1-c922b98cc311)

**Comment by mimixy** [comment]
> Retest this in newest update going out soon please

do i have to open a new issue? not sure how to make this one open again

**Comment by MysticalOS** [comment]
sadly that screen shot doesn't help. i need to see if classic changed the RP text for the FIRST yell (which is only partially visible in your shot)

**Comment by mimixy** [comment]
Doing BWL tomorrow, so will check and post a screenshot then.
I will say that ERA has received many big updates lately to its engine, to bring in line with HC/WOTLK clients.

**Comment by MysticalOS** [comment]
right, but it shouldn't have updated the texts, but we'll see

**Comment by mimixy** [comment]
![WoWScrnShot_010524_150455](https://github.com/DeadlyBossMods/DBM-Classic/assets/143043909/6c1f868e-0eee-4e44-887d-e55f6ba495fa)

**Comment by mimixy** [comment]
i streamed it too if you wanna see more. at the start of the stream i did /dbm to see version numbers https://www.twitch.tv/videos/2023060649

**Comment by MysticalOS** [comment]
Based on that it should work. Gonna ask obvious but did you turn option off?

**Comment by mimixy** [comment]
![WoWScrnShot_010524_201224](https://github.com/DeadlyBossMods/DBM-Classic/assets/143043909/cbd9bc4c-3166-4a33-8a5f-4dbb8555b458)

its the bottom setting right? it is worth nothing that this affects my entire guild and other guilds too. it's not just me. when I ask before vael if anyone can see the timer, no one can. 

is there a way i can trigger that timer without being in BWL? I wanna do some tests

**Comment by MysticalOS** [comment]
I have no idea. I cannot find any code reason why it doesn't work, unless everyone in raid is above 150ms

**Comment by mimixy** [comment]
nah there are for sure some under 150ms. prob most. i am 200ms myself. but have never had any issues seeing the timer.  does it matter who starts the fight?

otherwise i will keep running bwl and try different things like diff version of DBM. but i have a feeling its something blizz changed. we had a really big patch a couple months ago that broke a lot of things

**Comment by MysticalOS** [comment]
anyone over the default sync threshold if i think 150, doesn't send sync when yell happens, but if ANYONE is in range of yell when it happens that is under 150, it should sync to raid and start timer

**Comment by MysticalOS** [comment]
and yeah I know about 1.15.x but i don't think it broke comms, but who knows. it's really difficult to know where the problem is since something that worked, broke from a change I did not make :\

**Comment by mimixy** [comment]
i will mess around a bit more and report back. you can close this for now, will reply to you if i manage to work something out lol

**Comment by MysticalOS** [comment]
No it should remain open til fixed, even if it's not my fault it broke

**Comment by MysticalOS** [comment]
I found cause of breakage, it was my fault, but it wasn't something that broke recently, it's been broken for years. If it worked at any point, it was by sheer accident/miracle cause the code path for that combat timer was utterly busted and dead ended.

I've scrapped bad code and changed it to the more reliable and actually working RP timer object instead of combat timer. it should work in next update whenever that is.
